cfme/infrastructure/virtual_machines.py

```diff
@@ -1248,6 +1288,36 @@ def resetter(self, *args, **kwargs):
         self.view.reset_page()
 
 
+@navigator.register(InfraTemplateCollection, 'OrphanedAll')
+@navigator.register(InfraVmCollection, 'OrphanedAll')
+class OrphanedVms(CFMENavigateStep):
+    VIEW = OrphanedVmsAllView
+    prerequisite = NavigateToSibling('All')
+
+    def step(self, *args, **kwargs):
+        self.view.sidebar.vmstemplates.tree.click_path('All VMs & Templates', '<Orphaned>')
+
+    def resetter(self, *args, **kwargs):
+        if self.view.pagination.is_displayed:
```

Comment:
I don't think this should be done in the resetter for this navigation, as it will slow down navigation. Methods using the view can use `VMEntities` to easily iterate over the pages to get an entity.